### PR TITLE
Script to automatically create a Jenkins linux worker.

### LINKED ISSUE #4991: Create a script to provision new Jenkins linux workers
Currently workers 1, 4 and 5 are down, it would be good to have an automated way to recreate them.
This will probably be a combination of gcloud commands and a GCE provisioning script to be run once the machine has been created.

### PR DESCRIPTION
**Pull request by jtattermusch**
Fixes #4991.

### CODE REVIEW

File: tools/gce/linux_worker_init.sh [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+# Enable jenkins to use docker without sudo:
+sudo usermod -aG docker jenkins
+
+# Use "overlay" storage driver for docker
+# see https://github.com/grpc/grpc/issues/4988
+echo 'DOCKER_OPTS="${DOCKER_OPTS} --storage-driver=overlay"' | sudo tee --append /etc/default/docker
+
+# Install RVM
+# TODO(jtattermusch): why is RVM needed?
```
Comment (nicolasnoble) [inline tools/gce/linux_worker_init.sh]: The ruby builder, basically. There's in fact no need to have a dedicated machine for that. That's the only dependency I'd need for it.

**Comment by adelez** [comment]
I didn't see steps to install JRE.

File: tools/gce/linux_worker_init.sh [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+# Initializes a fresh GCE VM to become a jenkins linux worker.
+# You shouldn't run this script on your own, use create_linux_worker.sh
+# instead.
+
+set -ex
+
+sudo apt-get update
+
+# Install JRE
+sudo apt-get install -y openjdk-7-jre
```
Comment (jtattermusch) [inline tools/gce/linux_worker_init.sh]: @adelez these are the steps for JRE. I am pretty sure workers created by this script work fine, I created several of them over weekend.

**Comment by jtattermusch** [comment]
Ping.

**Pull request merged**